saleor/graphql/order/mutations/order_fulfill.py

```diff
@@ -105,6 +106,16 @@ class Meta:
         permissions = (OrderPermissions.MANAGE_ORDERS,)
         error_type_class = OrderError
         error_type_field = "order_errors"
+        webhook_events_info = [
+            WebhookEventInfo(
+                type=WebhookEventAsyncType.FULFILLMENT_CREATED,
+                description="A new fulfillment is created.",
+            ),
+            WebhookEventInfo(
+                type=WebhookEventAsyncType.ORDER_FULFILLED,
+                description="Order is fulfilled.",
```

Comment:
Isn't it like `OrderFulfill` can also trigger the event `FULFILLMENT_APPROVED`?
I mean this part: `saleor.order.actions.order_fulfilled`

Reply:
yes it does, and it already sent this event - adding missing webhook info